src/chat/chatter.py

```diff
@@ -141,6 +141,8 @@ def user_game(self, value):
 
         if self._user_game is not None:
             self._user_game.gameUpdated.connect(self.updateGame)
+            if self._user_game.state == GameState.PLAYING:
+                self._user_game.liveReplayAvailable.connect(self.updateGame)
```

Comment:
user_game is only set when a player's game changes, so if we join a game that's in another state (like OPEN), this will never get connected. We should always connect to it and check any extra conditions in the handler. Also make sure we disconnect from this signal when we change user_game again.

Reply:
hmm ... isn't joining a game, when it's OPEN, the usual case?
I only need that connection, if the game starts and that is a game.state change (we see chatter status change if a game starts .. so updateGame is called) ...?
So you suggest I lose the line 144?
I did a disconnect (like gameUpdated a few lines up), but that got me a lot of 'not connected' bug messages (mentioning the connect line, which step debug fine, so I don't know how to debug that and find out what's wrong. ... I guess delayed action on a connection makes it tricky)

Reply:
losing line 144, and putting the disconnect line back in done the trick. happy signaling ;-)